drake/matlab/solvers/NonlinearProgram.m

```diff
@@ -161,9 +161,12 @@
       obj.solver_options.snopt.MajorIterationsLimit = 1000;
       obj.solver_options.snopt.MinorIterationsLimit = 500;
       obj.solver_options.snopt.IterationsLimit = 10000;
-      obj.solver_options.snopt.MajorOptimalityTolerance = 1e-6;
-      obj.solver_options.snopt.MajorFeasibilityTolerance = 1e-6;
-      obj.solver_options.snopt.MinorFeasibilityTolerance = 1e-6;
+%       obj.solver_options.snopt.MajorOptimalityTolerance = 1e-6;
+%       obj.solver_options.snopt.MajorFeasibilityTolerance = 1e-6;
+%       obj.solver_options.snopt.MinorFeasibilityTolerance = 1e-6;
+      obj.solver_options.snopt.MajorOptimalityTolerance = 1e-3;
+      obj.solver_options.snopt.MajorFeasibilityTolerance = 1e-3;
+      obj.solver_options.snopt.MinorFeasibilityTolerance = 1e-3;
```

Comment:
Can you pass this in using solver_options rather than making a change to the defaults? Also, are the feasibility changes needed?